js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_ca.properties

```diff
@@ -123,3 +123,96 @@ error-person-name-invalid-character=El nom conté un caràcter no vàlid.
 user=Usuari
 applications=Aplicacions
 username=Nom d''usuari
+removeConsentSuccess=S'ha eliminat el consentiment amb èxit
+required=Requerit
+deviceActivity=Activitat dels aparells
+signOut=Tanca la sessió
+accept=Accepta
+addMultivaluedLabel=Afegeix {{fieldLabel}}
+avatar=Avatar
+usernamePlaceholder=Nom d'usuari o adreça electrònica
+deny=Denega
+selectALocale=Seleccioneu una llengua
+save=Desa
+resourceSharedWith_one=El recurs està compartit amb <0>{{username}}</0>
+resourceSharedWith_zero=Aquest recurs no està compartit.
+removeConsentError=No s'ha pogut eliminar el consentiment a causa de: {{error}}
+groupsListHeader=Capçalera de llista de grups
+privacyPolicy=Política de privacitat
```

Comment:
Observació: «Privacitat» o «Privadesa»? (qüestió d'estil).

Reply:
I'll switch to privadesa here and in another appearance, but termcat says they are interchangeable.